prow/config/inrepoconfig_test.go

```diff
@@ -81,469 +80,469 @@ func testDefaultProwYAMLGetter(clients localgit.Clients, t *testing.T) {
-					return fmt.Errorf("unexpected error: %w", err)
-				}
-				if n := len(p.Presubmits); n != 2 ||
-					p.Presubmits[0].Name != "hans" ||
-					p.Presubmits[1].Name != "kurt" {
-					return fmt.Errorf(`expected exactly two presubmits with names "hans" and "kurt" got %v`, p.Presubmits)
-				}
-				if n := len(p.Postsubmits); n != 2 ||
-					p.Postsubmits[0].Name != "karl" ||
-					p.Postsubmits[1].Name != "oli" {
-					return fmt.Errorf(`expected exactly two postsubmits with names "karl" and "oli", got %v`, p.Postsubmits)
-				}
-				if n := len(p.Presets); n != 2 ||
-					p.Presets[0].Labels["karl"] != "karlValue" ||
-					p.Presets[1].Labels["henning"] != "henningValue" {
-					return fmt.Errorf(`expected exactly two presets with labels "karl": "karlValue" and "henning": "henningValue", got %v`, p.Presets)
-				}
-				return nil
-			},
-		},
-		{
-			name: "Non-.yaml files under .prow directory are allowed",
-			baseContent: map[string][]byte{
-				".prow/one.yaml":     []byte(`presubmits: [{"name": "hans", "spec": {"containers": [{}]}}]`),
-				".prow/OWNERS":       []byte(`approvers: [approver1, approver2]`),
-				".prow/sub/two.yaml": []byte(`presubmits: [{"name": "kurt", "spec": {"containers": [{}]}}]`),
-				".prow/sub/OWNERS":   []byte(`approvers: [approver3, approver4]`),
-			},
-			validate: func(p *ProwYAML, err error) error {
-				if err != nil {
-					return fmt.Errorf("unexpected error: %w", err)
-				}
-				if n := len(p.Presubmits); n != 2 ||
-					p.Presubmits[0].Name != "hans" ||
-					p.Presubmits[1].Name != "kurt" {
-					return fmt.Errorf(`expected exactly two presubmit with name "hans" and "kurt", got %v`, p.Presubmits)
-				}
-				return nil
-			},
-		},
-		{
-			name: "Both .yaml and .yml files are allowed under .prow directory)",
-			baseContent: map[string][]byte{
-				".prow/one.yaml": []byte(`presubmits: [{"name": "hans", "spec": {"containers": [{}]}}]`),
-				".prow/two.yml":  []byte(`presubmits: [{"name": "kurt", "spec": {"containers": [{}]}}]`),
-			},
-			validate: func(p *ProwYAML, err error) error {
-				if err != nil {
-					return fmt.Errorf("unexpected error: %w", err)
-				}
-				if n := len(p.Presubmits); n != 2 ||
-					p.Presubmits[0].Name != "hans" ||
-					p.Presubmits[1].Name != "kurt" {
-					return fmt.Errorf(`expected exactly two presubmit with name "hans" and "kurt", got %v`, p.Presubmits)
-				}
-				return nil
-			},
-		},
-		{
-			name: "Basic happy path (presubmits, gerrit repo)",
-			baseContent: map[string][]byte{
-				".prow.yaml": []byte(`presubmits: [{"name": "hans", "spec": {"containers": [{}]}}]`),
-			},
-			validate: func(p *ProwYAML, err error) error {
-				if err != nil {
-					return fmt.Errorf("unexpected error: %w", err)
-				}
-				if n := len(p.Presubmits); n != 1 || p.Presubmits[0].Name != "hans" {
-					return fmt.Errorf(`expected exactly one presubmit with name "hans", got %v`, p.Presubmits)
-				}
-				return nil
-			},
-			repo: "repo/name",
-		},
+		// 		{
```

Comment:
Did you mean to leave all these tests commented out?

Reply:
not at all, will fix

Reply:
done